types/test/abi-test.ts

```diff
@@ -0,0 +1,219 @@
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { ABI } from 'caver-js'
+
+// $ExpectType typeof ABI
+ABI
+
+// $ExpectType ABI
+const abiCoder = new ABI()
+
+// $ExpectType string
+abiCoder.encodeFunctionSignature('myMethod(uint256,string)')
+// $ExpectType string
+abiCoder.encodeFunctionSignature({
+    name: 'myMethod',
+    type: 'function',
+    inputs: [
+        {
+            type: 'uint256',
+            name: 'myNumber',
+        },
+        {
+            type: 'string',
+            name: 'myString',
+        },
+    ],
+})
+
+// $ExpectType string
+abiCoder.encodeEventSignature('myEvent(uint256,bytes32)')
+// $ExpectType string
+abiCoder.encodeFunctionSignature({
+    name: 'myEvent',
+    type: 'event',
+    inputs: [
+        {
+            type: 'uint256',
+            name: 'myNumber',
+        },
+        {
+            type: 'bytes32',
+            name: 'myBytes',
+        },
+    ],
+})
+
+// $ExpectType string
+abiCoder.encodeParameter('uint256', '2345675643')
+// $ExpectType string
+abiCoder.encodeParameter('uint256', ['0xdf3234', '0xfdfd'])
+// $ExpectType string
+abiCoder.encodeParameter(
+    {
+        ParentStruct: {
+            propertyOne: 'uint256',
+            propertyTwo: 'uint256',
+            childStruct: {
+                propertyOne: 'uint256',
+                propertyTwo: 'uint256',
+            },
+        },
+    },
+    {
+        propertyOne: 42,
+        propertyTwo: 56,
+        childStruct: {
+            propertyOne: 45,
+            propertyTwo: 78,
+        },
+    }
+)
+
+// $ExpectType string
+abiCoder.encodeParameters(['uint256', 'string'], ['2345675643', 'Hello!%'])
+// $ExpectType string
+abiCoder.encodeParameters(['uint8[]', 'bytes32'], [['34', '434'], '0x324567fff'])
+// $ExpectType string
+abiCoder.encodeParameters(
+    [
+        'uint8[]',
+        {
+            ParentStruct: {
+                propertyOne: 'uint256',
+                propertyTwo: 'uint256',
+                ChildStruct: {
+                    propertyOne: 'uint256',
+                    propertyTwo: 'uint256',
+                },
+            },
+        },
+    ],
+    [
+        ['34', '434'],
+        {
+            propertyOne: '42',
+            propertyTwo: '56',
+            ChildStruct: {
+                propertyOne: '45',
+                propertyTwo: '78',
+            },
+        },
+    ]
+)
+
+// $ExpectType string
+abiCoder.encodeFunctionCall(
+    {
+        name: 'myMethod',
+        type: 'function',
+        inputs: [
+            {
+                type: 'uint256',
+                name: 'myNumber',
+            },
+            {
+                type: 'string',
+                name: 'myString',
+            },
+        ],
+    },
+    ['2345675643', 'Hello!%']
+)
+
+// $ExpectType string
```

Comment:
It looks like redundant lines 
from line number 137 to 142

Reply:
I removed